Server/src/main/java/org/xdi/oxauth/model/exception/AcrChangedException.java

```diff
@@ -7,6 +7,8 @@
 
 public class AcrChangedException extends Exception {
 
+    private boolean methodEnabled = true;
```

Comment:
I propose to rename it to `forceReAuthentication`, when I just looked in code I was not able to figure out at glance what `methodEnabled` means. what do you think?